pkgs/top-level/all-packages.nix

```diff
@@ -16773,6 +16773,8 @@ with pkgs;
 
   funnelweb = callPackage ../development/tools/literate-programming/funnelweb { };
 
+  func = callPackage ../applications/networking/cluster/func { };
```

Comment:
Nitpick, but can you add it before `funnelweb`

Reply:
Thanks, fixed